pytorch_lightning/trainer/supporters.py

```diff
@@ -177,3 +179,126 @@ def to_disk(self) -> None:
             # Write predictions for current file to disk
             with fs.open(filepath, "wb") as fp:
                 torch.save(outputs, fp)
+
+
+class AverageValueMeter:
+    '''
+    This class is taken from https://github.com/pytorch/tnt/blob/master/torchnet/meter/averagevaluemeter.py#L6
+    '''
```

Comment:
@MikkelAntonsen I see it here, as you moved it here in 530bddbd10c82d3adcf3931c1b24acb389255163